util/execdetails/execdetails.go

```diff
@@ -971,3 +1003,41 @@ func (e *RuntimeStatsWithCommit) formatBackoff(backoffTypes []fmt.Stringer) stri
 	sort.Strings(tpArray)
 	return fmt.Sprintf("%v", tpArray)
 }
+
+// FormatDurationForExplain uses to format duration, this function will prune precision before format duration.
+// Pruning precision is for human readability. The prune rule is:
+// 1. if the duration was less than 1us, return the original string.
+// 2. readable value >=10, keep 1 decimal, otherwise, keep 2 decimal. such as:
+//    9.412345ms  -> 9.41ms
+//    10.412345ms -> 10.4ms
+//    5.999s      -> 6s
```

Comment:
Is it possible to keep digits in zero? E.g. `6.00s` instead of `6s`. It can avoid some misunderstanding about the precision.

Reply:
I think this function is for better human readability, I don't think `6.00s` have better readability than `6s`

@SunRunAway What do you think?